src/Internal/Service/Level3Service.php

```diff
@@ -112,6 +104,29 @@ public function get_data_from_order( int $order_id ): array {
 			$level3_data['shipping_from_zip'] = $store_postcode;
 		}
 
+		/**
+		 * Filters the Level 3 data based on order.
+		 *
+		 * Example usage: Enables updating the discount based on the products in the order,
+		 * if any of the products are gift cards.
+		 *
+		 * @since 7.9.3
```

Comment:
```suggestion
		 * @since 8.0.0
```
FWIW, the next version of the plugin that this should be included in will be `8.0.0`.

Reply:
Thanks!